Makefile

```
@@ -33,7 +33,8 @@ deploy-shell: login-to-ecr
 
 .PHONY: build
 build:
-	docker-compose run tester make -C serverless-functions layer-codebuild STAGE=$(STAGE)
+	docker-compose run tester make -C serverless-functions build-without-publish STAGE=$(STAGE)
+	make -C serverless-functions publish-layer STAGE=$(STAGE)
```

Maybe we should separate building and publishing. up to you, its just a suggestion
```suggestion
.PHONY: publish
publish:
	make -C serverless-functions publish-layer STAGE=$(STAGE)
```